salt/apps/ocw/templates/webhook-publish.sh.jinja

```diff
@@ -113,18 +113,21 @@ cd /opt/ocw/ocw-www || error_and_exit "Can not cd to ocw-www"
 git_fetch_and_reset ocw-www $OCW_WWW_GIT_REF
 ocw_www_changed=$?
 
-# Pull ocw-course-hugo-starter
+# Temporarily modify ocw-www's go.mod to point at our local copy of ocw-hugo-themes
+cp go.mod go.mod.bak
```

Comment:
OK, that's good to know. Could you add an error handler, in case for some reason the file doesn't exist?

```
cp go.mod go.mod.bak || error_and_exit "go.mod does not exist yet"
```

Just to guard against the unexpected.

Reply:
Will do.  In the future, I'm hoping that a lot of this modification of the `go.mod` files will be unnecessary once we have proper versioning set up for `ocw-hugo-themes` and I can make a follow up PR to remove a lot of it.  I'm only adding in these `replace` directives at the moment to point at our locally cloned copy to be absolutely sure for the time being that we are using the `release` branch of `ocw-hugo-themes`.  Once we are properly tagging releases and everything is flowing, I think the `hugo mod get -u` commands should be able to just fetch the theme from github without having to explicitly do it outside of that.